vectordatasource/transform.py

```diff
@@ -729,16 +765,16 @@ def post_process_osm_zh(properties):
             if len(zh_Hant_fallback) == 0:
                 zh_Hant_fallback = names[0]
 
-    if 'name:zh-Hans' not in properties:
-        if len(zh_Hans_fallback) != 0:
+    if not properties.get('name:zh-Hans'):
+        if zh_Hans_fallback is not None and len(zh_Hans_fallback) != 0:
             properties['name:zh-Hans'] = zh_Hans_fallback
-        elif len(zh_Hant_fallback) != 0:
+        elif zh_Hant_fallback is not None and len(zh_Hant_fallback) != 0:
             properties['name:zh-Hans'] = zh_Hant_fallback
 
-    if 'name:zh-Hant' not in properties:
-        if len(zh_Hant_fallback) != 0:
+    if not properties.get('name:zh-Hant'):
+        if zh_Hant_fallback is not None and len(zh_Hant_fallback) != 0:
```

Comment:
I think this is the same as `if zh_Hant_fallback:`

Reply:
> I think this is the same as `if zh_Hant_fallback:`

indeed 👍